mempool/reactor.go

```diff
@@ -554,40 +554,43 @@ func newRedundancyControl(config *cfg.MempoolConfig) *redundancyControl {
 	}
 }
 
-func (rc *redundancyControl) controlLoop(memR *Reactor) {
-	for {
-		select {
-		case <-rc.adjustTicker.C:
-			// Compute current redundancy level and reset transaction counters.
-			redundancy := rc.currentRedundancy()
-			if redundancy < 0 {
-				// There were no transactions during the last iteration. Do not adjust.
-				continue
-			}
+func (rc *redundancyControl) adjustRedundancy(memR *Reactor) {
```

Comment:
This new function is now a refactoring. It doesn't change the logic. Up to you if you want to keep it.

Reply:
since we call it from `triggerAdjustment` I will leave it. Thanks for all the help on this!

Reply:
ah right, that's needed then 👍